docs/sources/flow/reference/components/module.string.md

```diff
@@ -0,0 +1,136 @@
+---
+title: module.string
+---
+
+# module.string
+
+`module.string` is a *Module loader* component. A *Module loader* is a Grafana Agent Flow 
+component which retreives a module and runs the components defined inside of it.
+
+*TODO: Add link to module concept page above once merged*
+
+## Usage
+
+```river
+module.string "LABEL" {
+	content   = CONTENT
+	arguments = {
+		argument1 = ARGUMENT1,
+		argument2 = ARGUMENT2,
+		...
+	}
+}
+```
+
+## Arguments
+
+The following arguments are supported:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`content`   | `secret`    | The contents of the module to load as a secret string. | | yes
+`arguments` | `map(any)`  | The values for the supported arguments in the module contents. | | no
+
+`content` is a string that contains all the arguments, exports and components for the module. 
```

Comment:
Can we say that `content` is a string which contains the configuration source of the module to load?